lib/topological_inventory/openshift/operations/source.rb

```diff
@@ -17,7 +17,7 @@ def initialize(params = {}, request_context = nil, metrics = nil)
 
         private
 
-        def connection_check(source_id)
```

Comment:
@lindgrenj6 The `source_id` variable is still being referenced as part of the log message in the `rescue` block.  Do we know the source_id so it can be properly logged?

Reply:
Yes, it's a method defined in the superclass here: https://github.com/RedHatInsights/topological_inventory-providers-common/blob/5c04f92462815485bffcb4f798bb82a490e28a82/lib/topological_inventory/providers/common/operations/source.rb#L21

This change happened when we moved the availability checking code into the providers-common gem.

Reply:
Do we also need to bump the messaging gem like we did here: https://github.com/RedHatInsights/topological_inventory-azure/pull/33/files

Reply:
Updated!